app/assets/stylesheets/components/_error-message.scss

```
@@ -0,0 +1,26 @@
+.error-message {
+  @extend .card;
+
+  margin-bottom: rem(1px);
+  border-color: $danger;
+
+  &__header {
+    @extend .card-header;
+
+    height: 2.5rem;
```

Let's turn all direct `rem` to `px`for consistency and better readability. There's one `1rem` at the end of this file.
```suggestion
    height: rem(40px);
```

---

Fixed in 46675db